blog/2017-08-28-react-native-monthly-3.md

```diff
@@ -0,0 +1,60 @@
+- We have landed HMR support into [`haul`](https://github.com/callstack-io/haul) and a bunch of other, cool stuff! Check out latest releases.
+- Conference is coming! Next month is all about React Native and Poland! Make sure to grab last tickets available [here](https://react-native.eu/)
+
+### Expo
+
+- Released support for installing npm packages on [Snack](https://snack.expo.io). Usual Expo restrictions apply -- packages can't depend on custom native APIs that aren't already included in Expo. We are also working on supporting multiple files and uploading assets in Snack. [Satyajit](https://github.com/satya164) will talk about Snack at [React Native Europe](https://react-native.eu/).
+- Released SDK20 with camera, payments, secure storage, magnetometer, pause/resume fs downloads, and improved splash/loading screen.
+- Continuing to work with [Krzysztof](https://github.com/kmagiera/react-native-gesture-handler) on [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler). Please give it a try, rebuild some gesture that you have previously built using PanResponder or native gesture recognizers and let us know what issues you encounter.
+- Experimenting with JSC debugging protocol, working on a bunch of feature requests on [Canny](https://expo.canny.io/feature-requests).
+
+### Facebook
+
+- Last month we discussed management of the GitHub issue tracker and that we would try to make improvements to address the maintainability of the project.
+- Currently, the number of open issues is holding steady at around 600, and it seems like it may stay that way for a while. In the past month, we have closed 690 issues due to lack of activity (defined as no comments in the last 60 days). Out of those 690 issues, 58 were re-opened for a variety of reasons (a maintainer committed to providing a fix, or a contributor made a great case for keeping the issue open).
+- We plan to continue with the automated closing of stale issues for the foreseeable future. We’d like to be in a state where every impactful issue opened in the tracker is acted upon, but we’re not there yet. We need all the help we can from maintainers to triage issues and make sure we don't miss issues that introduce regressions or introduce breaking changes, especially those that affect newly created projects. People interested in helping out can use the Facebook GitHub Bot to triage issues and pull requests. The new maintainers guide at facebook.github.io/react-native/docs/maintainers.html contains more information on triage and use of the GitHub Bot. Please add yourself to the issue task force at https://github.com/facebook/react-native/blob/master/bots/IssueCommands.txt and encourage other active community members to do the same!
+
```

Comment:
Shall FB update the status of the roadmap in monthly manner? :)

Reply:
What do you mean?

Reply:
Dear @hramos An uptodate roadmap. https://github.com/facebook/react-native/wiki/Roadmap